packages/modules/manage-versions/src/components/ManageVersions/ManageVersions.test.tsx

```diff
@@ -80,9 +80,9 @@ describe("ManageVersions", () => {
 
       expect(cells[2].textContent).toContain(mockedVersion.createdBy);
       expect(cells[3].textContent).toContain(mockedVersion.createdDateTime);
-      within(cells[4] as HTMLElement).getByTitle(
-        defaultStrings.updateNamedVersion
-      );
+      const actionsCell = cells[4] as HTMLElement;
+      const button = (actionsCell as HTMLElement).querySelector("button");
```

Comment:
cant you get by menu title?

Reply:
@veekeys Getting button by label text as now we are using `label `instead of `title`